ArucoDetectRos-Amir's_Changes.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python3
```

Comment:
```suggestion
#!/usr/bin/env python
```
You should use python2.7 when using rospy.